<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(235, 235, 222);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(79, 71, 71);
color: rgb(222, 222, 190);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string> Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreatePost">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 119, 70);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Create Post</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewProfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 119, 70);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>View you profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FollowButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 119, 70);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Follow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="feedButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 119, 70);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Feed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keysearchButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 119, 70);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Keyword search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="followEdit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>323</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(79, 71, 71);
color: rgb(222, 222, 190);
border: none;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(79, 71, 71);
color: rgb(222, 222, 190);
border: none;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
